docs/dev/aws_ipv6.md

```diff
@@ -0,0 +1,61 @@
+# AWS IPv6 Dev and Test Environment
+
+The installer includes code to enable an experimental IPv6 dev and test
+environment on AWS.  This is off by default and currently only intended for
+those working on enabling IPv6 in OpenShift.
+
+## Enabling the Environment
+
+To enable IPv6 in your AWS environment, set the following environment variable
+before running the installer:
+
+```bash
+    export OPENSHIFT_AWS_USE_IPV6=”true”
+```
+
+## AWS Network Environment
+
+AWS does not support a single-stack (IPv6 only) environment, but it does
+support a dual-stack (IPv4 and IPv6) environment, so that’s what is enabled
+here.  This is a summary of the changes to the network environment:
+
+* The VPC has IPv6 enabled and a `/56` IPv6 CIDR will be allocated by AWS.
+* Each Subnet will have an IPv6 `/64` subnet allocated to it.
+* All IPv4 specific security group rules have corresponding IPv6 rules created.
+* AWS Network Load Balancers (NLBs) do not support IPv6, so external API access
+  is still over IPv4.  AWS does not have a TCP load balancer that supports
+  IPv6, other than classic load balancers with EC2-Classic, and not EC2-VPC.
+  AWS Application Load Balancers supposedly support IPv6, but that would
+  require doing HTTPS load balancing for the API instead of just TCP load
+  balancing, so we just use the IPv4 NLBs.  API access within the cluster is
```

Comment:
Looks conflict with https://github.com/openshift/enhancements/pull/87/files#diff-7dbc8a4e63bd5b199e8c113fbc62609cR72, will use ELB for API or not?

Reply:
Ah yeah, the enhancement is out of date.  This doc is current.